configure.ac

```diff
@@ -51,4 +51,25 @@ PKG_CHECK_MODULES([libbase58],[libbase58])
 
 AC_CHECK_LIB([ws2_32], [strchr])
 
+#libcrypt necessary to build example.c with "make check"
+m4_ifdef([AM_PATH_LIBGCRYPT], [
+	AC_ARG_ENABLE([example],
```

Comment:
Maybe call it `--enable-examples` and give it a more general name, such as "Compile the examples". I think that's more in line with other projects, and future-proof in the case more examples are added.